test/functional/planar_3dtiles.js

```diff
@@ -0,0 +1,57 @@
+const assert = require('assert');
+
+describe('planar_3dtiles', function _() {
+    let result;
+    before(async () => {
+        result = await loadExample(
+            `http://localhost:${itownsPort}/examples/planar_3dtiles.html`,
+            this.fullTitle(),
+        );
+    });
+
+    it('should run', async () => {
+        assert.ok(result);
+    });
+
+    it('should pick the planar layer', async () => {
+        const layers = await page.evaluate(
+            () => view.pickObjectsAt({
+                x: 194,
+                y: 100,
+            })
+                .map(p => p.layer.id),
+        );
+
+        assert.ok(layers.indexOf('planar') >= 0);
+    });
+
+    it('should pick the correct building for 2012', async () => {
```

Comment:
The test failed because` timeout of 60000ms` exceeded.  So, it would have to position the camera differently to load less data.

Reply:
Yep good idea, done

Reply:
it's work